PWGJE/macros/AddTaskJetPP.C

```diff
@@ -0,0 +1,216 @@
+   if(trigger == AliVEvent::kAnyINT)
+      triggerName = "kAnyINT";
+   else if(trigger == AliVEvent::kAny)
+      triggerName = "kAny";
+   else if(trigger == AliVEvent::kINT7)
+      triggerName = "kINT7";
+   else if(trigger == AliVEvent::kMB)
+      triggerName = "kMB";
+   else if(trigger == AliVEvent::kEMC7)
+      triggerName = "kEMC7";
+   else if(trigger == AliVEvent::kEMCEJE)
+      triggerName = "kEMCEJE";
+   else if(trigger == AliVEvent::kEMCEGA)
+      triggerName = "kEMCEGA";
+   else 
+      triggerName = "kUNKW";
+
+ 
+   // #### DEFINE MANAGER AND DATA CONTAINER NAMES
+   AliAnalysisManager *manager = AliAnalysisManager::GetAnalysisManager();
+   if(!manager){
+      ::Error("AddTaskJetPP.C", "No analysis manager to connect to.");
+      return NULL;
+   }
+ 
+   TString containerNameSuffix("");
+   if(strcmp(containerSuffix,""))  containerNameSuffix = Form("_%s", containerSuffix);
+ 
+   TString myContName(""); //name of my container
+   myContName = Form("AnalysisAKTR%02dKTR%02d_%s_%s", 
+      TMath::Nint(jetAKTRadius*10),TMath::Nint(jetKTRadius*10), triggerName.Data(), containerNameSuffix.Data());
+
+   //_________________________________________________________
+   //  JET FINDER TASKS
+   enum AlgoType {kkt, kantikt};
+   enum JetType  {kfulljets, kchargedjets, kneutraljets};
+
+   if(jetAKTRadius < 0.1) return NULL;
+ 
+//__________________________________________________________________________________
+   //MC Tagger match jets
+   gROOT->LoadMacro("$ALICE_PHYSICS/PWGJE/EMCALJetTasks/macros/AddTaskEmcalJetTagger.C");
+   AliAnalysisTaskEmcalJetTagger* tagr = NULL;
+   //cout<<"jetfinderAkt="  <<jetfinderAkt<<endl;
+   if(ismc){
+      //cout <<"jetfinderAktmc =" <<jetfinderAktmc<<endl;
+      tagr = AddTaskEmcalJetTagger(jetfinderAkt,jetfinderAktmc,jetAKTRadius,
+                                                               "","", //nrhoBase, nrhoTa
+                                                                hybridtracks, //ntracks
+                                                                "", //nclusters
+                                                                "TPC",
+                                                                "V0A",//   centralityType,
+                                                                trigger,
+                                                                "",""  //trigClass, kEmcalTriggers
+                                                                );
+
+
+      tagr->SetJetTaggingType(AliAnalysisTaskEmcalJetTagger::kClosest); //GEN-REC JET MATCHING DONE HERE
+      tagr->SetJetTaggingMethod(AliAnalysisTaskEmcalJetTagger::kGeo);
+      tagr->SelectCollisionCandidates(trigger);
+      tagr->SetDebugLevel(0);
+      AliJetContainer *cont  = tagr->GetJetContainer(kContainerOne); //0
+      AliJetContainer *cont2 = tagr->GetJetContainer(kContainerTwo);//1
+      if(!cont) cout <<"AddTaskPP: Missing cont1"<<endl;
```

Comment:
Hi Dario,

This is in the add macro. While for AnalysisTasks I agree it is better to use AliDebug/Warning/Error(Stream), I think for the Add macro it can stay like this.

Best Regards

Markus